### ISSUE TITLE
Update language toggle component - Design

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#3605]
## Description
During the a11y review, Ryan raised a concern that only having bold text as the visual indicator may not be enough for visually impaired users.
The language toggle design needs updating to make the selected language more prominent. Here are some notes from the discussion between DST designers and Matt. 

- Keep it as inert if you are on current version of the page. Bold, black and underlined. 
- Don’t move or shift to accommodate the bar or icon. 
- If there is a WCAG reason icon is needed, okay with an icon. Follow side nav rules. Refer to USWDS implementation. Pad towards the text. If that doesn’t work, try other things. Go with bolder, thicker underline. As long as not blue. 

## Details
Dev ticket to make updates to address a11y issues: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3564

## Tasks
- [ ] Update the design of the language toggle component to add a bar or icon next to the selected language
- [ ] Review updated designs with DST designers and PO to get feedback
- [ ] Make any necessary changes 
- [ ] Post link to Figma design in #3564 

## Acceptance Criteria
- [ ] Language toggle designs have been updated

**Comment by LWWright7** [comment]
Per yesterday's meeting with @humancompanion-usds the design should utilize the colored bar to indicate the language that is currently "in use" similar to the USWDS side navigation component:
![Image](https://github.com/user-attachments/assets/491f9b9a-d954-4245-a4c8-2975fbf94be5)

**Comment by LWWright7** [comment]
@babsdenney @danbrady This is ready for [review](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=10077-912&t=Vj8zEvHBHZYUx3px-1)!

**Comment by danbrady** [comment]
@LWWright7 Similar to Barb's comment in Figma, I think we might want to explore a few options. I see you did what we discussed as a potential solution, but the vertical bar looks awkward to me. Not sure if it's because the active bar is shorter than the normal separator one? Or because English has a left bar when active but not when inactive? Or because the text is still blue like it's a link but it's not...? 

## For reference


### Currently published:

<img src="https://github.com/user-attachments/assets/3efced9d-d5e3-4fe1-9817-9ad4d4165b2c" width="200">

---

### Your proposed changes:

<img src="https://github.com/user-attachments/assets/1dc73ca5-19dd-411e-8ae8-c4d5f3e70d6f" width="200">

---


Honestly, I would expect it to simply look like this:

<img src="https://github.com/user-attachments/assets/c624d4f3-0c63-48b2-84ad-b94868271777" width="200" /><br><br>

Perhaps I'm misunderstanding the issue with this?

**Comment by LWWright7** [comment]
@rsmithadhoc are you suggesting something more like this?

![Image](https://github.com/user-attachments/assets/bea14918-7dc1-4e76-8e27-9b8e36241372)

**Comment by LWWright7** [comment]
Several other design considerations mentioned above now in [Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=10077-912&t=V7U3ALJB5HtQs952-1)

**Comment by rsmithadhoc** [comment]
> [@rsmithadhoc](https://github.com/rsmithadhoc) are you suggesting something more like this?
> 
> ![Image](https://github.com/user-attachments/assets/bea14918-7dc1-4e76-8e27-9b8e36241372)

> Several other design considerations mentioned above now in [Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=10077-912&t=V7U3ALJB5HtQs952-1)


@LWWright7 Yeah, I think keeping the current selection as a link and maintaining the link styling is good. So the option presented here or any of the options in Figma that do that are fine with me. Thanks!

**Comment by LWWright7** [comment]
@rsmithadhoc The design team met with @humancompanion-usds yesterday to discuss this matter and we took into account your comments in regards to the similarities of breadcrumb.  While we understand your position for a11y sake, @danbrady  pointed out this WCAG example breadcrumb [here](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/):

![Image](https://github.com/user-attachments/assets/659e36fc-b6c3-419b-ae67-a8a7c6d6ae21)

For the sake of better usability and user experience we are suggesting that the page/language that is active display to the user (or the language that is being displayed on the page) should differ from appearance from the other links that are active.  The user may be confused if they see what they think is a link and clicking on it does nothing, so we are suggesting that we follow the example set here by WCAG to make the current page text black in color.  

With that being said, we believe that one of these two designs is best suited for the task:
<img width="670" alt="Image" src="https://github.com/user-attachments/assets/b2a00f80-2143-4afb-8a2f-f3c77d9be3f1" />


We also wanted to tag @briandeconinck for your input and opinion on the matter at hand.

**Comment by LWWright7** [comment]
@babsdenney @danbrady  This one is ready for review!! 

@briandeconinck Would still love to hear your 2 cents on this one! :)

**Comment by rsmithadhoc** [comment]
> [@rsmithadhoc](https://github.com/rsmithadhoc) The design team met with [@humancompanion-usds](https://github.com/humancompanion-usds) yesterday to discuss this matter and we took into account your comments in regards to the similarities of breadcrumb. While we understand your position for a11y sake, [@danbrady](https://github.com/danbrady) pointed out this WCAG example breadcrumb [here](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/):
> 
> ![Image](https://github.com/user-attachments/assets/659e36fc-b6c3-419b-ae67-a8a7c6d6ae21)
> 
> For the sake of better usability and user experience we are suggesting that the page/language that is active display to the user (or the language that is being displayed on the page) should differ from appearance from the other links that are active. The user may be confused if they see what they think is a link and clicking on it does nothing, so we are suggesting that we follow the example set here by WCAG to make the current page text black in color.
> 
> With that being said, we believe that one of these two designs is best suited for the task: <img alt="Image" width="670" src="https://private-user-images.githubusercontent.com/146852077/397097257-b2a00f80-2143-4afb-8a2f-f3c77d9be3f1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3MDM1NjAsIm5iZiI6MTczNDcwMzI2MCwicGF0aCI6Ii8xNDY4NTIwNzcvMzk3MDk3MjU3LWIyYTAwZjgwLTIxNDMtNGFmYi04YTJmLWYzYzc3ZDliZTNmMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyMFQxNDAxMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZjgzNWUwNzBmYzc5MjVhYWQ5NjY1ZWIwODY0MTEwZTEzMzZjNWQxMDhjYzA3OWIxMWRkZmVmNDAzYmEwOGM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.5NEHUzadz5DeZFhJNRojliMaGPeGkiYbLo3TfEUjH3g">
> 
> We also wanted to tag [@briandeconinck](https://github.com/briandeconinck) for your input and opinion on the matter at hand.

If you feel strongly about that version, we can go with that. It's not a huge deal, so I don't think we need to drag out the discussion any longer.

However, I will point out that the example shared from W3C _is_ a link, it just doesn't look like one. 
<img width="959" alt="Image" src="https://github.com/user-attachments/assets/1d80ed26-c9a9-4a9e-b55a-c8611a25c46f" />

**Comment by danbrady** [comment]
Design looks good, @LWWright7 !

I tried all the combinations. It looks like when it's English and Tagalog only, with Tagalog active, the English link doesn't show: (As seen by the star)

<img width="772" alt="Image" src="https://github.com/user-attachments/assets/c5a8aa09-95b7-473d-a0b4-0893f33a9643" />

Also, I notice the "Show Espanol" property **does** display when it's the active language but "Show Tagalog" **does not** display when it is the active language. Ideally, it doesn't make sense to display the "Show [langauge]" property when that language is active. (Why would you hide the active language?) So if we can hide the "Show..." property for those specific cases, great. However, I can see Figma being finicky with it. So if "Show Espanol/Tagalog" needs to display even when they're the active languages, that's okay. I think we should be consistent with either way.

**Comment by babsdenney** [comment]
@LWWright7 This looks good, all the states worked well except the focus. The focus state is missing the yellow outline. 

<img width="386" alt="Image" src="https://github.com/user-attachments/assets/69ff8643-4610-4b57-85bf-476e69afcc4f" />

**Comment by LWWright7** [comment]
@babsdenney That's super odd....because if you look at the component itself, the focus state has the yellow outline there....

![Image](https://github.com/user-attachments/assets/bf2a7751-fdf6-4f70-82bf-cad5c945951a)

I have no idea why it would be doing this.  @danbrady do you have any clue why there would be this discrepancy?

**Comment by LWWright7** [comment]
Here's a video of it disappearing in real time.... 🤔

https://github.com/user-attachments/assets/3c73808f-16e7-46dd-9b3a-dc261f9fc288

@babsdenney @danbrady

**Comment by LWWright7** [comment]
This is carrying over to the next sprint. 

@babsdenney  @danbrady  - ready for review again.  Thanks!
